test/e2e/frontend/test_environment_os.js

```diff
@@ -1,14 +1,33 @@
 /* eslint-disable n/no-process-exit */
 'use strict'
 
-const app = require('./environments/standard')
+const smtp = require('./environments/smtp.js')
+const app = require('./environments/standard.js')
 
 ;(async function () {
     const PORT = 3001
+    const smtpConfig = {
+        smtpPort: process.env.SMTP_PORT || 1025,
+        webPort: process.env.SMTP_WEB_PORT || 8025
+    }
+
+    if (!process.env.NO_SMTP_SERVER) {
```

Comment:
Just want to be clear on this

The env var is called `NO_SMTP_SERVER` and is set to `true` in the GH Action setup, 

This `if` will evaluate if set to `false`?

Reply:
I relied on the fact that this env var will not be set anywhere else but CI and didn't take into account what values might be passed in. You are right though, it will evaluate to true no matter what value you'd pass.

I'll make it more concise